mentalHealthBot.py

```diff
@@ -0,0 +1,106 @@
+# # bot.py
+import json
+import os
+import discord
+import requests
+from discord.ext import commands
+import random
+
+# enter token for the server
+TOKEN = ''
```

Comment:
```suggestion
TOKEN = os.getenv('DISCORD_TOKEN')
```

Reply:
```suggestion
from dotenv import load_dotenv
load_dotenv()
TOKEN = os.getenv('DISCORD_TOKEN')
```

Reply:
I felt there was a small typo, just corrected it